Tools/Scripts/run-mvt-tests

```diff
@@ -30,317 +30,350 @@ if os.path.isdir(os.path.join(scripts_dir, 'webkitpy')):
 
 import argparse
 import json
-import subprocess
-import time
-
-import subprocess
-from subprocess import CalledProcessError
-
-from selenium import webdriver
+import logging
+import psutil
+import traceback
+import urllib3
+from random import randint
 from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.wpewebkit.options import Options
+from selenium.common.exceptions import TimeoutException
+
 
 top_level_directory = os.path.normpath(os.path.join(os.path.dirname(__file__), '..', '..'))
 
+_log = logging.getLogger(__name__)
+LOG_MESSAGE = 25
 TEST_SUITES = [
-    "codec-support-test",
-    "dash-html5-test",
-    "dash-shaka-test",
-    "dash-dashjs-test",
-    "hls-shaka-test",
-    "hls-hlsjs-test",
-    "hss-html5-test",
-    "hss-dashjs-test",
-    "progressive-html5-test",
+        "codec-support-test",
+        "dash-html5-test",
+        "dash-shaka-test",
+        "dash-dashjs-test",
+        "hls-shaka-test",
+        "hls-hlsjs-test",
+        "hss-html5-test",
+        "hss-dashjs-test",
+        "progressive-html5-test",
 ]
 
-COG_PLATFORMS = [
-    "drm",
-    "fdo",
-    "gtk4",
-    "headless",
-    "wl",
-]
-
-DEFAULT_BROWSER = "/usr/bin/cog"
-DEFAULT_WEBDRIVER = "/usr/bin/WPEWebDriver"
-DEFAULT_WEBDRIVER_ADDRESS = "0.0.0.0"
-DEFAULT_WEBDRIVER_PORT = "8088"
-DEFAULT_MVT_INSTANCE_ADDRESS = "https://mvt.rdkcentral.com"
-DEFAULT_PLATFORM = "headless"
-DEFAULT_TEST_SUITE = "dash-shaka-test"
-DEFAULT_CONFIGURATION = "release"
-
-def fatal(*args):
-    print()
-    print("Fatal:", end=' ')
-    print(*args)
-    sys.exit(1)
 
 def parse_args():
-    """Parse command line arguments"""
-
     parser = argparse.ArgumentParser(
         description="Run MVT suite with WebDriver.",
         epilog="""
-            This script connects to a running WPEWebDriver and fires up a MVT
+            This script uses WebDriver to automatically run the MVT
             test suite, collecting the results in a JSON file.
-
-            It also expects a running MVT instance in the environment
-            variable $MVT_INSTANCE_ADDRESS.
-
             For more info, check https://github.com/rdkcentral/mvt
             """,
     )
+    configuration = parser.add_mutually_exclusive_group(required=True)
+    configuration.add_argument('--debug', action='store_const', const='Debug', dest='configuration', help='Use a Debug build')
+    configuration.add_argument('--release', action='store_const', const='Release', dest='configuration', help='Use a Release build')
+    platform = parser.add_mutually_exclusive_group(required=True)
+    platform.add_argument('--gtk', action='store_const', dest='platform', const='gtk', help='Use the GTK port')
+    platform.add_argument('--wpe', action='store_const', dest='platform', const='wpe', help='Use the WPE port')
+    parser.add_argument("--extra-browser-args", help="Extra arguments to pass to the browser (run-minibrowser)")
```

Comment:
This may easily run into quoting awkwardness. How about handling `--` and taking the leftover command line arguments after that as the ones to be passed down to `run-minibrowser`? Many command line tools work that way.